propnet/symbols/properties/final_energy_per_atom.yaml

```diff
@@ -0,0 +1,9 @@
+name: final_energy_per_atom
+units:
+- 1.0
+- - [eV, 1.0]
+display_names: [Final Energy Per Atom]
+display_symbols: [E]
+dimension: 1
+test_value: 4.75
+comment: ''
```

Comment:
Add a comment here that this is a DFT-derived quantity to an arbitrary reference, so is not typically meaningful in isolation. Refer people to energy_above_hull_per_atom (or whatever we call it) as a more meaningful quantity

Another aside: these `test_values` ... the original intent here was that we'd use them to populate the graph with test data, but we're not actually using these anywhere correct? Do you think we should remove them? In my experience, as I've written properties, I've mostly just been picking a number out of thin air for this -- if that's what we're doing, it would probably be better just to generate numbers on the fly instead.

Reply:
So Anubhav indicated in one phone call that he'd like to be able to populate the graph with test data. I guess on those grounds we should keep it? I can ask him again in today's phone call. 

It should actually be trivial at this point to write the method to add test values to the graph. But also I don't understand the utility of adding test values considering each can apply to a completely different material / context. So together they seem meaningless.

Reply:
Yeah, I was thinking since we have these test data jsons anyway we can just pull data from there instead. It's not like all these test values are internally self consistent with each other anyway.

> But also I don't understand the utility of adding test values considering each can apply to a completely different material / context. So together they seem meaningless.

Yeah, exactly. One of the reasons I'm thinking they're a bit pointless and should just be removed.